apps/src/maze/CollectorGemCounter.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import color from '../util/color';
+import msg from './locale';
+
+const styles = {
+  container: {
+    display: 'inline-block',
+    fontFamily: '"Gotham 5r"',
+    paddingRight: 10,
+    pointerEvents: "none",
+    textAlign: 'right',
+    verticalAlign: 'top',
+  },
+  label: {
+    color: color.lighter_gray,
+    textTransform: 'capitalize'
+  },
+  hr: {
+    borderColor: color.lighter_gray,
+    borderWidth: 2,
+    margin: '0 auto',
+  },
+  gemImage: {
+    backgroundSize: "100%",
+    display: "inline",
+    padding: 5
+  },
+  checkmark: {
+    color: 'lightgreen',
+    fontSize: 18,
+    left: 5,
+    lineHeight: '25px',
+    position: 'relative',
+    top: 7,
+  },
+};
+
+export class CollectorGemCounter extends React.Component {
+  static propTypes = {
+    assetUrl: React.PropTypes.func.isRequired,
+    currentCollected: React.PropTypes.number.isRequired,
+    minRequired: React.PropTypes.number,
+  }
+
+  static defaultProps = {
+    minRequired: 1,
+  }
+
+  render() {
+    const showCheckmark = this.props.currentCollected >= this.props.minRequired;
+    const gemUrl = this.props.assetUrl('media/skins/collector/gem.png');
```

Comment:
You should be able to use `import assetUrl from '@cdo/apps/code-studio/assetUrl';` now!

https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/assetUrl.js